### DISCUSSION TITLE
Accessing GLPK options through PuLP on Google Colab Notebook

### DISCUSSION THREAD

**Discussion by SoudiptaC** [coin-or/pulp#353] [Q&A] [answered]
Hi All,

I have already asked this question on Stack Overflow but didn't get a reply; so, I am cross-posting it here:

[https://stackoverflow.com/questions/64486631/accessing-glpk-options-through-pulp-on-google-colab-notebook](https://stackoverflow.com/questions/64486631/accessing-glpk-options-through-pulp-on-google-colab-notebook)

In the sensitivity analysis, I want to calculate the allowable increases and decreases in the objective function coefficients and the right hand side parameters of the constraints. I could not figure out a way of doing it in PuLP's native solver; hence, I am looking at GLPK.
 
Could you please help me out?

Thanks!

**Comment by pchtsp** [comment, marked as answer]
I paste the answer I gave in SO:

The options argument to pass to GLPK_CMD need to have no spaces, so:

```prob.solve(GLPK(msg=True, options=['--ranges', 'sensitivity.txt']))```

Then it works. With your cases, GLPK gave an error without solving the problem saying:

```
Invalid option '--ranges sensitivity.txt'; try glpsol --help
```

**Reply by SoudiptaC** [reply]
Thanks! This works!
